web/src/main/webapp/ui/ui.ts

```diff
@@ -63,7 +63,7 @@ export class HtmlString {
     }
 }
 
-export type ViewKind = "Table" | "Histogram" | "2DHistogram" | "Heatmap" |
+export type ViewKind = "Table" | "Histogram" | "PrivateHistogram" | "2DHistogram" | "Heatmap" |
```

Comment:
Is this used anywhere?

Reply:
Hm, I guess it's not used right now. Will we need this eventually? ViewKind is used in several places, not sure which of them we will need to adapt for the private histogram as well.